authors/apps/authentication/serializers.py

```diff
@@ -121,6 +121,39 @@ def validate(self, data):
         }
 
 
+class ResetSerializer(serializers.ModelSerializer):
+    """Get users email"""
+    email = serializers.EmailField()
+
+    def validate_email(self, data):
+        email = data.get('email', None)
+
+    class Meta:
+        model = User
+        fields = ('email',)
+
+
+class PasswordSerializer(serializers.ModelSerializer):
+    """Validates Password"""
+    password = serializers.RegexField(
```

Comment:
Since this is not different from what was used in the RegistrationSerializer, can we keep our code DRY?